src/lib/orders.ts

```diff
@@ -45,6 +45,11 @@ export const getOrderStatusInfo = ({ status }: { status: string }) => {
         statusIconColor: `var(--color-error)`,
       };
     }
+    case AbacusOrderStatus.partiallyCanceled.rawValue:
```

Comment:
nit. we can probably just do sequential

```
case AbacusOrderStatus.cancelled.rawValue:
case AbacusOrderStatus.partiallyCanceled.rawValue:
``` 

on line 42 since they're supposed to look the same!